library/Ivoz/Kam/Infrastructure/Kamailio/TrunksClient.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Ivoz\Kam\Infrastructure\Kamailio;
+
+use Ivoz\Core\Infrastructure\Domain\Service\Gearman\Client\XmlRpcTrunksRequestInterface;
+use Ivoz\Kam\Domain\Service\TrunksClientInterface;
+use Psr\Log\LoggerInterface;
+
+class TrunksClient implements TrunksClientInterface
+{
+    protected $rpcClient;
+    protected $germanClient;
+    protected $logger;
+
+    public function __construct(
+        RpcClient $rpcClient,
+        XmlRpcTrunksRequestInterface $germanClient,
+        LoggerInterface $logger
+    ) {
+        $this->rpcClient = $rpcClient;
+        $this->germanClient = $germanClient;
+        $this->logger = $logger;
+    }
+
+    public function reloadDialplan()
+    {
+        return $this->germanClient->send(
+            self::DIALPLAN_RELOAD_ACTION
+        );
+    }
+
+    public function reloadDispatcher()
+    {
+        return $this->germanClient->send(
+            self::DISPATCHER_RELOAD_ACTION
+        );
+    }
+
+    public function reloadLcr()
+    {
+        return $this->germanClient->send(
+            self::LCR_RELOAD_ACTION
+        );
+    }
+
+    public function reloadTrustedPermissions()
+    {
+        return $this->germanClient->send(
+            self::PERMISSIONS_TRUSTED_RELOAD_ACTION
+        );
+    }
+
+    public function reloadAddressPermissions()
+    {
+        return $this->germanClient->send(
+            self::PERMISSIONS_ADDRESS_RELOAD_ACTION
+        );
+    }
+
+    public function reloadUacReg()
+    {
+        return $this->germanClient->send(
+            self::UAC_REG_RELOAD_ACTION,
+            true
+        );
+    }
+
+    /**
+     * @param int $companyId
+     * @return int
+     */
+    public function getCompanyActiveCalls(int $companyId)
+    {
+        $payload = ['activeCallsCompany'];
+        if (!is_null($companyId)) {
+            $payload[] = $companyId;
+        }
+
+        return $this->getActiveCalls($payload);
+    }
+
+    /**
+     * @param int $brandId
+     * @return int
+     */
+    public function getBrandActiveCalls(int $brandId)
+    {
+        $payload = ['activeCallsBrand'];
+        if (!is_null($brandId)) {
+            $payload[] = $brandId;
+        }
+
+        return $this->getActiveCalls($payload);
+    }
+
+    /**
+     * @return int
+     */
+    public function getPlatformActiveCalls()
+    {
+        return $this->getActiveCalls([
+            'activeCallsBrand'
+        ]);
+    }
+
+    /**
+     * @param array $payload
+     * @return int
+     */
+    private function getActiveCalls(array $payload)
+    {
+        $response = $this->sendRequest(
+            'dlg.profile_get_size',
```

Comment:
I would create a constant for this RPC method like all the other actions have

Reply:
I like it :)